ovl_util/logs.py

```diff
@@ -11,14 +11,15 @@
 from typing import TextIO
 
 from ovl_util.config import load_config
-from root_path import root_dir
+# TODO: log_dir and config_path used in this file should be parameterized instead of hardcoded based on root_dir
+root_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
```

Comment:
also

Reply:
Take a look at the `# TODO: ` comment above this line.